src/network/networkprotocol.h

```diff
@@ -870,7 +873,12 @@ enum ToClientCommand
 			f32 center_weight_power
 	*/
 
-	TOCLIENT_NUM_MSG_TYPES = 0x64,
+	TOCLIENT_PLAYER_POS = 0x64,
```

Comment:
Calling the new packet `TOCLIENT_PLAYER_POS` seems misleading because normal player position updates are not done via the new packet, but via `TOCLIENT_MOVE_PLAYER`.

Perhaps the new packet could be called `TOCLIENT_MOVE_PLAYER_REL` (relative)?

Reply:
Renamed.